app/presenters/document_presenter.rb

```diff
@@ -37,13 +37,26 @@ def to_json
   def details
     {
       body: GovspeakPresenter.present(document.body),
+      headers: headers,
```

Comment:
Should this be in the `.tap` so that we don't add it if there are no `headers` at all - I don't think we have spec for that scenario.